opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -246,7 +250,10 @@ impl<R: RuntimeChannel> SpanProcessor for BatchSpanProcessor<R> {
         let result = self.message_sender.try_send(BatchMessage::ExportSpan(span));
 
         if let Err(err) = result {
-            global::handle_error(TraceError::Other(err.into()));
+            otel_debug!(
+                name: "BatchSpanProcessor.Export.SendResultError",
```

Comment:
this is confusing. This is the `on_end` method.. Given this is BatchProcessor, this will only queue up spans on `on_end` and export occurs asynchronously. The name "BatchSpanProcessor.Export.SendResultError" seem to indicate that some sort of Export is going on here, which is incorrect/misleading, as export never occurs on `on_end` call for BatchSpanProcessor. Also "SendResultError" is not reflecting anything going on here...

Reply:
Good catch. Have named it to `BatchSpanProcessor.OnEnd.ExportQueueingFailed`.